front/app/utils/streams/index.ts

```diff
@@ -995,7 +879,106 @@ class Streams {
 
     return await Promise.all(promises);
   }
-}
 
```

Comment:
This is mostly the same as `.get` but with everything we don't need stripped away